deps/spidershim/test/value.cc

```diff
@@ -921,6 +947,361 @@ TEST(SpiderShim, ExternalStringResourceDestructorCalled) {
   EXPECT_TRUE(externalOneByteStringResourceDestructorCalled);
 }
 
+// Begin code copied from THREADED_TEST(StringWrite) in test-api.cc.
```

Comment:
FWIW in other tests that I have copied, I just included one line indicating the name of the original test.  I don't think this verbosity is needed!  I'll fix after merging.